### ISSUE TITLE
[MINOR] Zoom in/out in network graph with touch screen

### ISSUE THREAD

**Issue by shamiv** [org-roam/org-roam-ui#286]
I love org-roam-ui and it has been an important part of my workflow for quite some time now.

I was wondering if it would be easy/possible to navigate through the network graph using the touch screen of my laptop.

Right now, the gestures "work" but:

1. When zooming using the touch screen (two fingers pinching), the resolution goes down (or actually, the original pre-zoom resolution stays the same, which means that "zooming in" shows the images at a lower resolution).
2. When I use the mouse to zoom in to a particular part of the graph, and then zoom in with the touch screen, I can use one finger to move up/down left/right, but only within the part of the graph that I zoomed into using the mouse.

In other words, the touch screen gestures (in contrast to the mouse gestures) seem to be detached from the graph building?

I'm on a Lenovo Thinkpad 440s, I run Ubuntu 22.04, Emacs 28.1 and I'm up to date with org-roam-ui.
